spec/models/druid_version_zip_spec.rb

```diff
@@ -295,4 +318,35 @@
       expect(dvz.zip_storage.to_s).to eq '/tmp'
     end
   end
+
+  describe '#check_moab_version_readability!' do
+    let(:dvz) { described_class.new(druid, version, 'spec/fixtures/storage_root01/sdr2objects') }
+    let(:v1_moab_files) do
+      %w[
+        spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/content/eric-smith-dissertation-augmented.pdf
+        spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/content/eric-smith-dissertation.pdf
+        spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/contentMetadata.xml
+        spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/descMetadata.xml
+        spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/identityMetadata.xml
+        spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/provenanceMetadata.xml
+        spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/relationshipMetadata.xml
+        spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/rightsMetadata.xml
+        spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/technicalMetadata.xml
+        spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/versionMetadata.xml
+        spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/manifests/fileInventoryDifference.xml
+        spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/manifests/manifestInventory.xml
+        spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/manifests/signatureCatalog.xml
+        spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/manifests/versionAdditions.xml
+        spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/manifests/versionInventory.xml
+      ]
+    end
+
+    # note: if this test is dropped or altered, revisit the #create_zip! tests in this spec w.r.t.
```

Comment:
Why would this test be dropped or altered?

Reply:
not totally sure, and i was a bit on the fence about including this comment (and the corresponding one on line 105).

i don't think this test is strictly necessary in terms of code coverage results, and so i can imagine it getting deleted accidentally, or intentionally as cruft/overkill, and then not getting noticed if the PR review is cursory.  i wanted to avoid that.  it's also testing some behavior that seems unnecessary at first glance, so it was easy to imagine aggressive code pruning removing it.

if that scenario seems unlikely to you, and you think this adds too much noise, i can remove the comment.

Reply:
I think this is confusing. I think this comment could apply to almost every test case so I'm not clear why we need to call this out in particular.

Reply:
fair enough, got rid of it